release-team/role-handbooks/communications/README.md

```diff
@@ -35,6 +35,7 @@ The expected time investment for both leads and shadows are as follows:
 There is a channel on the Kubernetes Slack workspace, `release-comms`, which is used by the communications release team to coordinate their efforts. If you're on the communications team, or applying to be, then it would be advantageous to follow along with the conversations.
 
 ## Release Milestone Activities
+Ensure to have the release team leads and shadows in CC during the communication with CNCF.
```

Comment:
```suggestion
Ensure to have the release team leads and shadows are notified while communicating with the CNCF.
```

Reply:
Please defer reviews until my comment is addressed: https://github.com/kubernetes/sig-release/pull/986#issuecomment-585802554